event-adapter/README.md

```diff
@@ -0,0 +1,181 @@
+# 1. Create a cluster ensuring that Stackdriver Logging API are enabled (both reading and writing)
+```sh
+mkdir -p $GOPATH/src/k8s.io
+cd $GOPATH/src/k8s.io
+git clone git@github.com:kubernetes/kubernetes.git
+cd kubernetes
+git checkout 9aef242a4c1e423270488206670d3d1f23eaab52
+sed -i 's/logging-write/https:\/\/www.googleapis.com\/auth\/logging.admin/g' cluster/gce/config-common.sh
+make quick-release
+./cluster/kube-up.sh
+```
+You should create “k8s.io” folder to host the project. Then, clone kubernetes project on git. Checkout to the desired version and compile it.
```

Comment:
Since this is not a documentation for Kubernetes, you don't have to explain the commands in detail. I suggest rather:

"Download Kubernetes and create a cluster with Stackdriver Logging enabled:
```sh (...) ```

Reply:
Done